### PR TITLE
Editor/hdrp create own path

### PR DESCRIPTION
**Pull request by thomas-tu**
**Goal of this PR:**
Make sure Legacy render pipeline and HDRP run on different path for picking.

### CODE REVIEW

File: Content/Shader/ProBuilderCG.cginc [hunk 1]
```diff
@@ -27,3 +27,22 @@ inline float4 UnityObjectToClipPosWithOffset(float3 pos)
     ret *= lerp(.99, .95, ORTHO);
     return mul(UNITY_MATRIX_P, ret);
 }
+
+inline float4 GetPickerColor(float4 pos, float2 texcoord1)
```
Comment (karl-) [inline Content/Shader/ProBuilderCG.cginc]: This is just expanding the vertices to form a billboard, not setting a color.

File: Content/Shader/FacePickerHDRP.shader [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        Pass
+        {
+            Name "Base"
+
+            CGPROGRAM
+            #pragma vertex vert
+            #pragma fragment frag
+            #include "UnityCG.cginc"
+
+            float4 _Tint;
```
Comment (karl-) [inline Content/Shader/FacePickerHDRP.shader]: Unused

File: Content/Shader/VertexPickerHDRP.shader [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            Name "Vertices"
+
+CGPROGRAM
+            #pragma vertex vert
+            #pragma fragment frag
+            #include "UnityCG.cginc"
+            #include "ProBuilderCG.cginc"
+
+            // Is the camera in orthographic mode? (1 yes, 0 no)
+            #define ORTHO (1 - UNITY_MATRIX_P[3][3])
```
Comment (karl-) [inline Content/Shader/VertexPickerHDRP.shader]: I think this is already defined in `ProBuilderCG.cginc`

**Comment by thomas-tu** [comment]
> @thomas-tu I can make these changes if you're busy, just let me know

Yes, I'll let you handle it. Thanks!

**Pull request merged**